package.json

```diff
@@ -2,8 +2,12 @@
   "name": "xstate-websockets",
   "private": true,
   "description": "WebSocket server and client packages built with XState",
-  "scripts": {},
+  "scripts": {
+    "postinstall": "husky install"
```

Comment:
This command automatically installs the `git` hook for `husky` after an install (`pnpm i`).